Sources/DistributedCluster/Pattern/WorkerPool.swift

```diff
@@ -196,9 +229,21 @@ extension WorkerPool {
         /// The worker pool will terminate itself if all of its static workers have terminated.
         /// You may death-watch the worker pool in order to react to this situation, e.g. by spawning a replacement pool,
         /// or gracefully shutting down your application.
-        public static func `static`(_ workers: [Worker]) -> Selector {
-            .init(underlying: .static(workers))
-        }
+        case `static`([Worker])
+    }
+
+    public enum Strategy {
+        /// Simple random selection on every target worker selection.
+        case random
+
+        /// Round-robin strategy which attempts to go "around" known workers one-by-one
+        /// giving them equal amounts of work. This strategy is NOT strict, and when new
+        /// workers arrive at the pool it may result in submitting work to previously notified
+        /// workers as the round-robin strategy "resets".
+        ///
+        /// We could consider implementing a strict round robin strategy which remains strict even
+        /// as new workers arrive in the pool.
+        case simpleRoundRobin
     }
```

Comment:
Again we should make a struct with an underlying enum inside it, and make static functions to create instances. Otherwise by making the enum public we'll never be able to add new cases without a major release.